packages/astro/src/core/config/schema.ts

```diff
@@ -541,17 +541,23 @@ export const AstroConfigSchema = z.object({
 				.object({
 					driver: z.string(),
 					options: z.record(z.any()).optional(),
-					cookieName: z.string().optional(),
-					cookieOptions: z
-						.object({
-							domain: z.string().optional(),
-							path: z.string().optional(),
-							expires: z.string().optional(),
-							maxAge: z.number().optional(),
-							httpOnly: z.boolean().optional(),
-							sameSite: z.string().optional(),
-							secure: z.boolean().optional(),
-							encode: z.string().optional(),
+					cookie: z
+						.union([
```

Comment:
What happens if an integration calls `updateConfig()` with jsut the cookie name? Would it  override the user options?

Reply:
Yeah. I'm not sure why they would, but the integration would need to take care to not override user settings